### ISSUE TITLE
$WP_DEBUG doesn't exist; use the constant WP_DEBUG instead

### ISSUE THREAD

**Issue by pjohanneson** [duosecurity/duo_universal_wordpress#65]
## Description

In `class-duouniversal-utilities.php`, the `duo_debug_log()` function uses `$WP_DEBUG`, a variable which WordPress doesn't use.

## Expected Behavior

The constant `WP_DEBUG` should be used instead.

## Workarounds

Change:

```php
function duo_debug_log( $message ) {
	global $WP_DEBUG;
	if ( $WP_DEBUG ) {
		error_log( 'Duo debug: ' . $message );
	}
}
```

to:

```php
function duo_debug_log( $message ) {
	if ( defined( 'WP_DEBUG' ) && WP_DEBUG ) {
		error_log( 'Duo debug: ' . $message );
	}
}
```

**Comment by pjohanneson** [comment]
Ah, it seems I'm not the only one who noticed. There's a fix in merge request #62.

**Comment by AaronAtDuo** [comment]
Fixed in #62, we will close this issue once we release.
